files/scripts/update-cluster.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+set -eu
+
+function patchCluster {
+  local configLocation=$1
+  local authToken=$2
+  echo "Patching cluster"
+  waitService "http://graphdb-cluster-proxy:7200/proxy/ready" "$authToken"
```

Comment:
what if the external proxy is not deployed?

Reply:
Is it a valid scenario to deploy a cluster and not have an external proxy? 
This job executes only if you've requested more than 1 node and will always execute last because of it's weight if by this point there is no cluster something has gone terribly wrong and you are probably aware of it.